setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import find_packages, setup
+
+
+setup(
+    version='0.1.0',
+    name='incuna-pagination',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[],
+    description='Utilities for paginated views.',
+    author='Incuna Ltd',
+    author_email='admin@incuna.com',
+    url='https://github.com/incuna/incuna-pagination',
+)
```

Comment:
It would be good to add [trove classifiers](http://python-packaging-user-guide.readthedocs.org/en/latest/distributing/#classifiers).